user/cmd/main.go

```diff
@@ -2,22 +2,41 @@ package main
 
 import (
 	"github.com/GrabItYourself/giys-backend/lib/logger"
+	"github.com/GrabItYourself/giys-backend/lib/postgres"
 	"github.com/GrabItYourself/giys-backend/user/config"
+	"github.com/GrabItYourself/giys-backend/user/pkg/handler"
+	"github.com/GrabItYourself/giys-backend/user/pkg/repository"
 	"github.com/gofiber/fiber/v2"
 	"github.com/pkg/errors"
 )
 
 func main() {
+	// Config
 	conf := config.InitConfig()
+
+	// Logger
 	logger.InitLogger(&conf.Log)
 
-	app := fiber.New()
+	// Repository
+	pg, err := postgres.New(&conf.Postgres)
+	if err != nil {
+		logger.Fatal(errors.Wrap(err, "Can't initialize postgres").Error())
+	}
+	repo := repository.New(pg)
 
+	// Handlers
+	handlers := handler.NewHandler(repo)
```

Comment:
Should we use gRPC instead of REST? I thought that the request would be from Client — (REST) —> API Gateway — (gRPC) —> Services

REST itself has huge overhead compared with gRPC

Reply:
will change in next PRs